src/smokeTest/java/uk/gov/hmcts/reform/draftstore/ApplicationSmokeTest.java

```diff
@@ -1,25 +1,72 @@
 package uk.gov.hmcts.reform.draftstore;
 
-import io.restassured.RestAssured;
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.junit.Test;
-import org.springframework.boot.actuate.health.Status;
-import org.springframework.http.HttpStatus;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.draftstore.response.Draft;
 
-import static org.hamcrest.Matchers.equalTo;
+import java.util.Optional;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
 public class ApplicationSmokeTest extends SmokeTestSuite {
 
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private static final String SAMPLE_DRAFT_JSON = "{\"some\": \"draft\"}";
```

Comment:
may be worth renaming to 'sample document' as this is not the actual draft json that's sent to the API.
or just create an instance of `CreateDraft` model here.

Reply:
Fair point. Done.